ci/pipelines/deploy-to-staging.yml

```diff
@@ -1845,8 +1852,19 @@ jobs:
         trigger: true
         passed: [deploy-adminusers-to-staging]
       - get: pay-ci
+      - task: parse-staging-release-tag
+        file: pay-ci/ci/tasks/parse-staging-release-tag.yml
+        input_mapping:
+          git-release: endtoend-git-release
```

Comment:
So these two lines are where the problem is. The  `parse-staging-release-tag.yml` task is currently expecting `inputs` of two folders: `pay-ci` and `ecr-repo`. That's not what we're providing it here 😸

Reply:
@katstevens I made this change
 ```
task: parse-staging-release-tag
        file: pay-ci/ci/tasks/parse-staging-release-tag.yml
        input_mapping:
          ecr-repo: adminusers-ecr-registry-staging
```

with a new error 

```
selected worker: ip-10-1-10-50
An error occurred: ReferenceError: release is not defined
```